PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.h

```diff
@@ -388,6 +389,12 @@ class AliAnalysisTaskGammaConvV1 : public AliAnalysisTaskSE {
     Float_t                           fEta_PhotonML;                              
     Float_t                           fPhi_PhotonML;                              
     Float_t                           fR_PhotonML;                              
+    Float_t                           fCosThetaPA_PhotonML;
```

Comment:
In principle we dont need to stream all your variables. You could indicate that they should not be streamed be commenting //! (see https://alice-doc.github.io/alice-analysis-tutorial/analysis/streamer.html)